standards/SPIFFE_Federation.md

```diff
@@ -307,6 +307,56 @@ As described in the [Trustworthiness of the Bundle Endpoint Server](#74-trustwor
 
 Finally, it should be noted that the "links" in this chain are formed by individual HTTPS request operations (against the various bundle endpoint servers in the chain), and these operations are likely occurring at different times. SPIFFE bundle endpoint clients should be logging these HTTPS request operations, and administrators should take care to preserve these logs for future forensic analysis if necessary.
 
+### 7.7 Processing claims asserted by foreign trust domains
+
+SPIFFE Federation allows workloads in one trust domain to authenticate SVIDs, and all the claims therein, issued by other trust domains. By an "authentic" claim, we mean specifically that it can be safely assumed that the foreign trust domain authority _asserted_ the claim. This is not the same thing as saying that the claim is _true_, or that it is _correctly interpreted_.
+
+Operators should use caution and good judgement in evaluating if and how they should process claims asserted in SVIDs from foreign trust domains. The recommended posture is not to process foreign claims that have not been evaluated accoring to the framework described here (i.e. default deny), then assert exceptions for specific claims from specific trust domains to be allowed if they pass muster.
+
+It bears repeating that implementations should never process any claims from SVIDs that fail authentication (except in a forensic context).
+
+#### 7.7.1 SVID Claims
+
+For the purpose of this discussion, a *claim* is an assertion about some attribute of the subject of an SVID, or about the SVID itself. The canonical SPIFFE claim is the SPIFFE ID, which is a claim about the subject of the SVID. An example of a claim about the SVID itself is the expiration time.
+
+SVID formats (to date including X.509 and JWT) afford flexibility to encode additional claims, including claims that have not been evaluated by any standards body.
```

Comment:
What is an example of a claim that has been evaluated by a standards body?

I think what this paragraph is trying to say is:
SVID formats (e.g. X.509 and JWT) intentionally include the ability to include claims which are not specified by the SVID or underlying format (e.g. X.509, JWT) specifications.

Reply:
No, here I'm thinking of things like X.520 that define standard attributes https://www.itu.int/rec/T-REC-X.520-201610-S

These can be encoded in X.509 SANs as General Names.